components/elm/src/biogeophys/HydrologyNoDrainageMod.F90

```diff
@@ -248,12 +248,12 @@ subroutine HydrologyNoDrainage(bounds, &
 
 #ifndef _OPENACC
        if (use_betr) then
-          call ep_betr%BeTRSetBiophysForcing(bounds, col_pp, veg_pp, 1, nlevsoi, waterstate_vars=waterstate_vars, &
-             waterflux_vars=waterflux_vars, soilhydrology_vars = soilhydrology_vars)
+          call ep_betr%BeTRSetBiophysForcing(bounds, col_pp, veg_pp, 1, nlevsoi, waterstate_vars=col_ws, &
+             waterflux_vars=col_wf, soilhydrology_vars = soilhydrology_vars)
 
           call ep_betr%DiagAdvWaterFlux(num_hydrologyc, filter_hydrologyc)
 
-          call ep_betr%RetrieveBiogeoFlux(bounds, 1, nlevsoi, waterflux_vars=waterflux_vars)
+          call ep_betr%RetrieveBiogeoFlux(bounds, 1, nlevsoi, waterflux_vars=col_wf)
```

Comment:
Same question as in my previous comment, can the WaterStateType and WaterFluxType be removed here?